src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/DotNetFormatTests.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.DotNet.SourceBuild.SmokeTests;
 
 public class DotNetFormatTests : SdkTests
 {
+    private const string BaselineSubDir = "DotNetFormatTests";
```

Comment:
Would it make sense to use nameof(DotNetFormatTests) for these?  This applies throughout the PR.